raiden/tests/unit/test_upgrade.py

```diff
@@ -48,3 +50,41 @@ def test_upgrade_manager_restores_backup(tmp_path):
     assert state_change_record.data is not None
     assert not old_db_filename.exists()
     assert Path(str(old_db_filename).replace('_log.db', '_log.backup')).exists()
+
+
+def test_sequential_version_numbers(tmp_path):
```

Comment:
nitpick: Maybe just add docstring here explaining what the test does? Thanks to our call today I know exactly what you want to test but perhaps explain for the others too?

Something like:
```
"""
Test that when there are multiple DB upgrades to be applied each one is ran and only once.
"""
```

Reply:
done